### DISCUSSION TITLE
Allure Report is not picking Test case which is failed due to "After Each " Hook

### DISCUSSION THREAD

**Discussion by Animeshkr29** [Shelex/cypress-allure-plugin#80] [Q&A] [answered]
Hi Development Team 

while Running i Notice Allure report not able to pick failed test which is failed in "After Each " hook , 
but same Failure is showing properly in Mocha Report , 

Can any one please help , due to this issue we have count Mismatch in Allure Cypress report

**Comment by Shelex** [comment]
Hi @Animeshkr29 

Actually I am not able to reproduce this issue. 
Could you please provide repository with reproducible example and specify your environment details (os, node version, cypress version, cypress plugin version, if there are any other plugins used)?

**Reply by Animeshkr29** [reply]
Hi Shelex , Thanks for reply and my Apologies to late reply from my end , please find my answer below 

1. **Regarding Repository Location**  : My Repo is under Private Network and can't share in open forum . How ever I gone through your repository : https://github.com/Shelex/cypress-allure-plugin-example/blob/master/cypress/integration/examples/hooks.spec.js
my test is almost same , could you please try below Describe block once 

### **Test Case 1** : Test Should Fail in AfterEach only
describe('Allure hooks', () => {

    beforeEach(() => {
        cy.log('from beforeEach');
    });
    afterEach(() => {
        cy.log('Do any Operation which mandatory Fail Here ');
    });

    it('Let this Test Pass but make Sure it should fail in AfterEach', () => {
        cy.log('from test');
    });
});

### **Test Case 2** : Test Should fail only in BeforeEach 
describe('Allure hooks', () => {

    beforeEach(() => {
        cy.log('Do any Operation which mandatory Fail Here ');
    });
    afterEach(() => {
        cy.log('Let this have commend which Pass');
    });

    it('Let this test block should not run and make Sure it should fail in BeforeEach', () => {
        cy.log('from test');
    });
});

### **Plugin Version Details** 
     "@shelex/cypress-allure-plugin": "^2.8.3",
     "allure-commandline": "^2.13.8",
     "cypress": "7.7.0",
     "mocha": "^8.1.1",
    "mocha-junit-reporter": "^1.23.0",
    "mocha-multi-reporters": "^1.1.7",
    "mochawesome": "^6.1.1",
    "mochawesome-merge": "^4.0.3",
    "mochawesome-report-generator": "^5.1.0"

other Plugins are same like upload doc, aws , sql etc .

i am running in Linux box With Docker .

i am Running my test from Command Line where i am Passing --env allure=true along with Mocha , Junit reporter as well .

### For Example : 
npx cypress run --spec cypress/integration/*.spec.js --browser chrome --headed --env allure=true,grepTags= --reporter mocha-multi-reporters --reporter-options configFile=testrail-config.json

Let me know any other Details you Need , and please help me to solve this issue since due to this issue we have test case count discrepancy with Mochawesome report

**Comment by Animeshkr29** [comment]
Any Input ? Are you able to reproduce the reported Issue ?

**Reply by Shelex** [reply]
Hi @Animeshkr29.
As discussions are reserved just for questions I have converted it to issue https://github.com/Shelex/cypress-allure-plugin/issues/97 where I suggest to track that as I was not notified about your reply here. Hope will have time next week to check it.

**Comment by Shelex** [comment, marked as answer]
Seems like issue https://github.com/Shelex/cypress-allure-plugin/issues/97 is closed, marking this thread as answered

**Reply by mihirsasmal** [reply]
Is there any configuration needs to be update for this one ?
 I have the latest version and, in my report, test which fails due to before each hook is not showing.

**Reply by Shelex** [reply]
Hi.
No configuration is required, it should just work out of the box.
If in your case it does not - please open an issue, specify plugin and cypress versions, if possible - include repo with reproducible example.
